Tasks/Gradle/Gradle.ps1

```diff
@@ -105,9 +105,15 @@ if ($jdkPath)
 
 $buildRootPath = $cwd
 $wrapperDirectory = Split-Path $wrapperScript -Parent
-$reportDirectoryName = [guid]::NewGuid()
+$reportDirectoryName = "ReportDirectory_84B7D86C-F08D-4B65-8D37-742D8B02D27B"
```

Comment:
should we keep the name shorter as ashwini has done in her PR. to avoid the pathtoolong issue